libagent/gpg/__init__.py

```diff
@@ -226,13 +238,18 @@ def run_agent(device_type):
     p.add_argument('-v', '--verbose', default=0, action='count')
     p.add_argument('--server', default=False, action='store_true',
                    help='Use stdin/stdout for communication with GPG.')
-
-    p.add_argument('--pin-entry-binary', type=str, default='pinentry',
-                   help='Path to PIN entry UI helper.')
-    p.add_argument('--passphrase-entry-binary', type=str, default='pinentry',
-                   help='Path to passphrase entry UI helper.')
-    p.add_argument('--cache-expiry-seconds', type=float, default=float('inf'),
-                   help='Expire passphrase from cache after this duration.')
+    if device_type.package_name() == 'onlykey-agent':
+        p.add_argument('-sk', '--skey', type=int, metavar='SIGN_KEY',
```

Comment:
@onlykey @romanz the -sk feature did not make in #337. Was that on purpose?

Reply:
Not on purpose, I just wanted to simplify #337.
PRs are welcome :)